.github/workflows/continuous-integration-cypress.yml

```diff
@@ -12,12 +12,18 @@ env:
 
 jobs:
   cypress-tests:
-    if: github.event.workflow_run.conclusion == 'success' && github.event.workflow_run.inputs.environment != 'Production'
-    environment: ${{ github.event.workflow_run.inputs.environment }}
+    if: github.event.workflow_run.conclusion == 'success'
+    strategy:
+      matrix:
+        include:
+          - environment: dev
+          - environment: staging
+    environment:
+      name: ${{ matrix.environment }}
```

Comment:
How does does the matrix get set though?
Anyway, good to know about it. I had no idea how to solve this particular problem.

Reply:
It's my understanding that it will do two parallel runs, one against `dev` and one against `staging` just by including these two `environment` matrix values